app/adapters/goldencobra/article_adapter.rb

```diff
@@ -0,0 +1,66 @@
+require "addressable/uri"
+
+module Goldencobra
+  class ArticleAdapter
+    class << self
+
+
+      # Alias method for find_by_url
+      # @param url [String] "http://www.IkusEI.de/afasdf/?asAAAa=12"
+      # @param options = {} Options of method find_by_url
+      #
+      # @return [Integer] ArticleID
+      def find(url, options = {})
+        find_by_url(url, options)
+      end
+
+      # find an article ID by an given url
+      # @param url [String] "http://www.IkusEI.de/afasdf/?asAAAa=12"
+      # @param follow_redirection [Boolean] default: false
+      #
+      # If you use this method in an controller,
+      # the controller should redirect to a redirection.
+      #
+      # If you use this method somewhere else, you should activate
+      # follow_redirection:true to be sure you get the right article
+      #
+      # @return [Integer] ID of an article
+      def find_by_url(url, follow_redirection: false)
+        url_to_search = cleanup_url(url)
+        url_to_search = follow_redirections(url_to_search) if follow_redirection
+        Goldencobra::ArticleUrl.where(url: url_to_search).pluck(:article_id).first
+      end
+
+      # find an url by an given ID
+      # @param search_id [Integer]
+      #
+      # @return [Array] list of urls
+      def find_by_id(search_id)
```

Comment:
Do you think `find_url_by_id` would be more friendly to use, because it is more clear what the return of this method is?